backend/protos/mgmt/v1alpha1/anonymization.proto

```diff
@@ -27,8 +33,15 @@ message AnonymizeManyResponse {
   repeated AnonymizeManyErrors errors = 2;
 }
 
+message TransformerMapping {
+  // JQ Expression or Field Path to apply the transformation to
+  string expression = 1;
+  // Configuration of Transformer to apply
+  TransformerConfig transformer = 2;
```

Comment:
I suppose we could use the required field here too!

```

// If `required` is true, the field must be populated. A populated field can be
--
// described as "serialized in the wire format," which includes:
//
// - the following "nullable" fields must be explicitly set to be considered populated:
//   - singular message fields (whose fields may be unpopulated/default values)
//   - member fields of a oneof (may be their default value)
//   - proto3 optional fields (may be their default value)
//   - proto2 scalar fields (both optional and required)
// - proto3 scalar fields must be non-zero to be considered populated
// - repeated and map fields must be non-empty to be considered populated
//
// ```proto
// message MyMessage {
//   // The field `value` must be set to a non-null value.
//   optional MyOtherMessage value = 1 [(buf.validate.field).required = true];
// }
// ```
```